packages/utils/src/api/polkadot.api.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import LRU from 'lru-cache';
+
+const tenMin = 10 * 60 * 1000;
+
+const cache = new LRU<string, Promise<ApiPromise>>({
+  max: 20,
+  ttl: tenMin,
+  updateAgeOnGet: true,
+  ttlAutopurge: true,
```

Comment:
Why do we need this parameter? isn't it better to let the expired items drop when new items are added?

Reply:
The idea was that if you leave a tab open and do nothing connections will drop in 10 min to not keep them open.

Reply:
Ok